gcs-snippets/README.md

```diff
@@ -0,0 +1,58 @@
+# Cloud Storage snippets
```

Comment:
opt_nit: perhaps name the directory "storage-snippets" instead? I know acronym use is discouraged at least in Google documentation/code.

Reply:
I'd prefer this too because most users wouldn't know what gcs is. Or even using google-storage-snippets to be more precise.

Reply:
renamed!